pkgs/development/libraries/geos/default.nix

```diff
@@ -1,22 +1,33 @@
-{ lib, stdenv, fetchurl }:
+{ lib
+, stdenv
+, fetchurl
+, cmake }:
 
 stdenv.mkDerivation rec {
   pname = "geos";
-  version = "3.9.1";
+  version = "3.10.2";
 
   src = fetchurl {
     url = "https://download.osgeo.org/geos/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-fmMFB9ysncB1ZdJJom8GoVyfWwxS3SkSmg49OB1+OCo=";
+    sha256 = "sha256-ULvFmaw4a0wrOWLcxBHwBAph8gSq7066ciXs3Qz0VxU=";
   };
 
+  nativeBuildInputs = [ cmake ];
+
   enableParallelBuilding = true;
 
```

Comment:
```suggestion
```
already set by cmake

Reply:
Change made with latest force push. Thanks for catching!